src/main/java/com/epam/grid/engine/provider/utils/slurm/job/SacctCommandParser.java

```diff
@@ -0,0 +1,125 @@
+                    .overSubscribe(jobData.get(7))
+                    .jobId(Integer.parseInt(jobData.get(8)))
+                    .name(jobData.get(9))
+                    .comment(jobData.get(10))
+                    .timelimit(jobData.get(11))
+                    .minMemory(jobData.get(12))
+                    .reqNodes(jobData.get(13))
+                    .command(jobData.get(14))
+                    .priority(Double.parseDouble(jobData.get(15)))
+                    .qos(jobData.get(16))
+                    .reason(jobData.get(17))
+                    .stateCompact(jobData.get(19))
+                    .userName(jobData.get(20))
+                    .reservation(jobData.get(21))
+                    .wckey(jobData.get(22))
+                    .excNodes(jobData.get(23))
+                    .nice(jobData.get(24))
+                    .sct(jobData.get(25))
+                    .execHost(jobData.get(26))
+                    .account(jobData.get(27))
+                    .cpus(Integer.parseInt(jobData.get(28)))
+                    .nodes(Integer.parseInt(jobData.get(29)))
+                    .dependency(jobData.get(30))
+                    .arrayJobId(Integer.parseInt(jobData.get(31)))
+                    .groupId(Integer.parseInt(jobData.get(32)))
+                    .socketsPerNode(jobData.get(33))
+                    .coresPerNode(jobData.get(34))
+                    .threadsPerSocket(jobData.get(35))
+                    .arrayTaskId(jobData.get(36))
+                    .timeLeft(jobData.get(37))
+                    .timeUsed(jobData.get(38))
+                    .nodeList(parseListFromString(jobData.get(39)))
+                    .contiguous(Integer.parseInt(jobData.get(40)))
+                    .partition(jobData.get(41))
+                    .priorityLong(Long.parseLong(jobData.get(42)))
+                    .nodeListReason(parseListFromString(jobData.get(43)))
+                    .startTime(convertStringToTimeEntry(jobData.get(44)))
+                    .state(jobData.get(45))
+                    .uid(Integer.parseInt(jobData.get(46)))
+                    .submissionTime(convertStringToTimeEntry(jobData.get(47)))
+                    .licenses(jobData.get(48))
+                    .coreSpec(jobData.get(49))
+                    .schedNodes(jobData.get(50))
+                    .workDir(jobData.get(51))
+                    .build();
+        } catch (final NumberFormatException ex) {
+            throw new GridEngineException(HttpStatus.INTERNAL_SERVER_ERROR,
+                    FAILED_TO_PARSE_JOB_DATA, ex);
+        }
+    }
+
+    private static List<String> parseListFromString(final String stringList) {
+        return List.of(stringList.split(COMMA));
+    }
+
+    private static LocalDateTime convertStringToTimeEntry(final String dateTime) {
+        return !dateTime.equals("N/A")
+                ? DateUtils.tryParseStringToLocalDateTime(dateTime, DateTimeFormatter.ofPattern(SLURM_DATE_FORMAT))
```

Comment:
You have a suitable `DateUtils.tryParseStringToLocalDateTime(String, String)` method, so this string could be more compact

Reply:
fixed